### PR TITLE
fill fee_limit only if nonce

### PR DESCRIPTION
**Pull request by huazhouwang**
## What does this implement/fix? Explain your changes.
仅在 fee_limit 为空时候填充

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
https://github.com/OneKeyHQ/TaskHub/issues/1564

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

File: electrum_gui/common/provider/chains/eth/provider.py [hunk 1]
```diff
@@ -70,10 +70,13 @@ def fill_unsigned_tx(self, unsigned_tx: UnsignedTx) -> UnsignedTx:
 
             if data:
                 payload["data"] = data
-            if token_address or self.geth.is_contract(to_address):
+
+            if not fee_limit and (token_address or self.geth.is_contract(to_address)):
```
Comment (taimanhui) [inline electrum_gui/common/provider/chains/eth/provider.py:74]: `fee_limit is None`，`token_address is not None`

这里的逻辑之前没有关注，是不是说没有指定limit时转给非合约地址的话直接使用21000就够了不需要估算，其它情况下就估算

**Review by taimanhui — APPROVED** [review]

File: electrum_gui/common/provider/chains/eth/provider.py (hunk 1 above)
Comment (huazhouwang) [reply electrum_gui/common/provider/chains/eth/provider.py:74]: 嗯，这种情况就是转给普通地址主币，只需要 21000 就够了

**Comment by huazhouwang** [comment]
@shuaiLiWang 有空帮忙 review 下

**Review by shuaiLiWang — APPROVED** [review]

**Comment by taimanhui** [comment]
@huazhouwang 不要依赖github的update branch，会多出来个merge commit

**Comment by huazhouwang** [comment]
> @huazhouwang 不要依赖github的update branch，会多出来个merge commit

了解，下次注意

**Pull request merged**